components/ws-manager-bridge/src/redis/publisher.ts

```diff
@@ -16,21 +15,19 @@ import {
     RedisWorkspaceInstanceUpdate,
     WorkspaceInstanceUpdatesChannel,
 } from "@gitpod/gitpod-protocol";
+import { Redis } from "ioredis";
 
 @injectable()
 export class RedisPublisher {
```

Comment:
Is not copying the publisher in two places the main problem?

Reply:
It is, but I'm first moving the Redis Client, and then moving the publisher. To move the publisher, I'll also need to add prometheus metrics to do the `gitpod-db` package which will be a bigger change so trying to slice it up